common/web/types/src/consts/virtual-key-constants.ts

```diff
@@ -136,8 +136,27 @@ export const USVirtualKeyMap: number[][] = [
   [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_LBRKT, k.K_RBRKT, k.K_BKSLASH ],
   // [caps] A S D F G H J K L ; ' [enter]
   [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE ],
+  // [shift] Z X C V B N M , . / [shift] *=oE2
+  [ k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH ],
```

Comment:
Note that some "US" style mongrel keyboards have a 102nd key...

Reply:
is the key on the right hand side? we may need to handle it as a separate hardware layout then

Reply:
I don't think we need to handle it as part of the LDML keyboard spec. In Core, we should make the 102nd key inherit the behaviour of the Windows US keyboard (emit `\` and `|`) when used with a `"us"` hardware layout ldml keyboard, so it will still work okay.